seed/src/event.rs

```diff
@@ -21,36 +21,52 @@ use librad::{meta::project::ProjectInfo, net::peer::PeerApi, peer::PeerId, uri::
 
 use crate::{guess_user, Error, Project, Signer};
 
+/// A peer with optional user metadata.
+#[derive(Clone, Debug)]
+pub struct User {
+    pub peer_id: PeerId,
+    pub urn: Option<RadUrn>,
+    pub name: Option<String>,
+}
```

Comment:
Kind of weird that this is called `User` when the doc says "peer". Based on the fields, I think this should be called `Peer`, because a user without a URN or a name is not a user.

Reply:
`Peer` doesn't hae a name, so it cuts both ways.

Reply:
:thinking: